### ISSUE TITLE
[gfs] conjugate of tail gives incorrect result

### ISSUE THREAD

**Issue by tayral** [TRIQS/triqs#248]
Hi,
the formula for the conjugate of the tail is incorrect:
````
g = GfImFreq(indices=[0],beta=10.)
g2 = GfImFreq(indices=[0],beta=10.)

g << inverse(iOmega_n-3)
g2<< g.conjugate()

print g.tail, g2.tail
```` 
gives
````
tail/tail_view: min/smallest/max = -1 1 8
 ...  Order -1 = 
[[(0,0)]]
 ...  Order 0 = 
[[(0,0)]]
 ...  Order 1 = 
[[(1,0)]]
 ...  Order 2 = 
[[(3,0)]]
 ...  Order 3 = 
[[(9,0)]]
 ...  Order 4 = 
[[(27,0)]]
 ...  Order 5 = 
[[(81,0)]]
 ...  Order 6 = 
[[(243,0)]]
 ...  Order 7 = 
[[(729,0)]]
 ...  Order 8 = 
[[(2187,0)]] tail/tail_view: min/smallest/max = -1 1 8
 ...  Order -1 = 
[[(0,-0)]]
 ...  Order 0 = 
[[(0,-0)]]
 ...  Order 1 = 
[[(1,-0)]]
 ...  Order 2 = 
[[(3,-0)]]
 ...  Order 3 = 
[[(9,-0)]]
 ...  Order 4 = 
[[(27,-0)]]
 ...  Order 5 = 
[[(81,-0)]]
 ...  Order 6 = 
[[(243,-0)]]
 ...  Order 7 = 
[[(729,-0)]]
 ...  Order 8 = 
[[(2187,-0)]]
````

**Comment by krivenko** [comment]
Is it the same as #67?

**Comment by parcollet** [comment]
It is the same and was assigned to @mferrero 
And the fix is not correct : it will not work properly for real omega Gf.

**Comment by tayral** [comment]
OK. How to access the info imag vs. real frequency from within the tail 
class?

**Comment by parcollet** [comment]
No way, the pb has to be solved otherwise, 
specialize conj of the gf, make an option in the conj of the tail, etc...
We discussed that with @mferrero , I don't know if he has made the patch or not.
He was working on it.

**Comment by tayral** [comment]
closed as duplicate of #67
